test/test_files.py

```diff
@@ -194,6 +194,10 @@ def test_components_works_on_relative(self):
         p = 'a/b/c'
         a = ['a', 'b', 'c']
         self.assertEqual(util.components(p), a)
+    def test_forward_slash(self):
```

Comment:
Travis will likely complain about the missing blank line above this one. :smiley: